.github/workflows/build-docker-image.yml

```diff
@@ -70,7 +70,7 @@ jobs:
         with:
           # list of Docker images to use as base name for tags
           images: |
-            us-docker.pkg.dev/zealous-zebra/zebra/${{ inputs.image_name }}
+            us-docker.pkg.dev/${{ vars.GCP_PROJECT }}/zebra/${{ inputs.image_name }}
```

Comment:
How can we make sure we have replaced all the hard-coded strings with variables?
Was this PR created using a tool like `fastmod` or `grep`?

Reply:
I used my VSCode replacement tool, but I had to anyways validate most of the mass replacement before applying it as not all of them applied _as-is_ like the `IMAGE_NAME` (which I'll improve in the next refactor)